toolkit/tools/pkg/imagecustomizerlib/bootcustomizer.go

```diff
@@ -61,14 +101,14 @@ func (b *BootCustomizer) AddKernelCommandLine(extraCommandLine []string) error {
 
 	combinedArgs := GrubArgsToString(extraCommandLine)
 
-	if b.isGrubMkconfig {
+	if b.bootConfigType == BootConfigTypeGrubMkconfig {
```

Comment:
Add a comment for the `BootConfigTypeUki` saying why nothing needs to be done.

Similar comment elsewhere.